package.json

```diff
@@ -99,7 +102,8 @@
     "standard": "standard '{bin,examples,lib,test}/**/*.js'",
     "nyc": "NODE_TLS_REJECT_UNAUTHORIZED=0 nyc --reporter=text-summary mocha",
     "mocha": "NODE_TLS_REJECT_UNAUTHORIZED=0 mocha",
-    "test": "npm run standard && npm run nyc"
+    "test": "npm run standard && npm run nyc",
```

Comment:
standard should go or be replaced with lint

Reply:
Why, though?

Reply:
_If_ this PR replaces standard by lint, then this should happen. (It would crash now because the dependency was removed.)

Reply:
Which dependency was removed in which branch?